server/db/product/product.js

```diff
@@ -65,6 +65,19 @@ ProductSchema.methods = {
   toggleActivation: function toggleActivation () {
     this.isActivated = !this.isActivated;
     return this;
+  },
+
+  rentalUpdate: function rentalUpdate (update) {
+      if (update.username !== undefined){
+      this.rentSchedule[0] = {
```

Comment:
I have a question, Brent! :)
Are we going to overwrite 0 index element of rentSchedule array everytime, not adding new schedule?